src/main/java/frc/robot/commands/BangBangSubsystemCommand.java

```diff
@@ -0,0 +1,41 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.DriveSub;
+
+public class BangBangSubsystemCommand extends Command{
+    private final DriveSub subsystem;
+    private final double setpoint;
+    private final double power = 0.5;
+
+    public BangBangSubsystemCommand(DriveSub subsystem, double setpoint) {
+        this.subsystem = subsystem;
+        this.setpoint = setpoint;
+        addRequirements(subsystem);
+    }
+
+    public void setPoint(double setpoint) {
+
+    }
+
+    @Override
+    public void execute() {
+        double currentPosition = subsystem.getPosition();
+        if (currentPosition < setpoint) {
+            subsystem.setSpeed(power); // drive forward
+        } else {
+            subsystem.setSpeed(-power); // drive backward
+        }
+    }
+
+    @Override
+    public void end(boolean interrupted) {
+        subsystem.setSpeed(0); // stop the motor
+    }
+
+    @Override
+    public boolean isFinished() {
+        double currentPosition = subsystem.getPosition();
+        return Math.abs(currentPosition - setpoint) < .5; // tolerance
```

Comment:
Now that everything is in rotations, a tolerance of 0.5 rotations is too high. You can divide this by 360 or set it to some arbitrarily lower value.

Reply:
changed the tolerance to 1 / 360

Reply:
It now uses integer division, so 1/360 = 0. To fix this, you can use 1.0/360 or Units.rotationsToDegrees().

Reply:
I think I said the wrong thing. I meant degreesToRotations, not rotationsToDegrees. You can use 1.0/360 or Units.degreesToRotations(1).